src/Search/Gambit/SuspendedGambit.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Suspend\Search\Gambit;
+
+use Carbon\Carbon;
+use Flarum\Search\AbstractRegexGambit;
+use Flarum\Search\AbstractSearch;
+use Flarum\User\Search\UserSearch;
+use Flarum\User\UserRepository;
+use LogicException;
+
+class SuspendedGambit extends AbstractRegexGambit
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected $pattern = 'is:suspended';
+
+    /**
+     * @var \Flarum\User\UserRepository
+     */
+    protected $users;
+
+    /**
+     * @param \Flarum\User\UserRepository $users
+     */
+    public function __construct(UserRepository $users)
+    {
+        $this->users = $users;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function apply(AbstractSearch $search, $bit)
+    {
+        if (! $search->getActor()->can('suspend')) {
+            return false;
+        }
+
+        return parent::apply($search, $bit);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function conditions(AbstractSearch $search, array $matches, $negate)
+    {
+        if (! $search instanceof UserSearch) {
+            throw new LogicException('This gambit can only be applied on a DiscussionSearch');
```

Comment:
I think you meant to write UserSearch here

Reply:
Ah that's awkward :see_no_evil:, I'll push a fix shortly. Thanks for catching this!